### PR TITLE
currency exchange route

### PR DESCRIPTION
**Pull request by Akashdeep-Patra**
image

image

image

### CODE REVIEW

File: pages/currency-exchange/currency-exchange.module.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.exchange_rates {
+  margin: 5%;
+}
+
+@media only screen and (max-width: 1200px) {
+  /*Tablets [601px -> 1200px]*/
+}
```
Comment (sumitd94) [inline pages/currency-exchange/currency-exchange.module.css:7]: Why is this needed when this does not contain any css in it?
same for line no 14-16

File: pages/currency-exchange/index.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import personData from 'mock/person.json';
+import NavBar from '@components/NavBar';
+import Head from 'next/head';
```
Comment (sumitd94) [inline pages/currency-exchange/index.js:4]: NIT: This should be placed at line no 2

File: styles/Home.module.css [hunk 3]
```diff
@@ -25,7 +29,7 @@
   border: none;
 }
 .card {
-  min-width: 20em;
+  /* min-width: 20em; */
```
Comment (sumitd94) [inline styles/Home.module.css:32]: If this is not needed, please remove it instead of commenting it.

File: components/exchange-rate-row/index.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      <span className={exchange_rate_row__arrow}>&#8667;</span>
+      <div className={second_currency_wrapper}>
+        <div className={second_currency_wrapper__name}>{second_currency}</div>
+        <img
+          src={second_image}
+          alt={second_currency}
+          className={second_currency_wrapper__image}
+        />
+      </div>
+    </div>
```
Comment (sumitd94) [inline components/exchange-rate-row/index.js:44]: Can see a common pattern here, can we generalised this one? may be create a `currency` component and call it here 2 times with necessary props?

File: components/exchange-rate-row/exchange-rate-row.module.css [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+.first_currency_wrapper__name {
+  margin: auto 0;
+}
+.first_currency_wrapper__image {
+  margin: auto 0;
+}
+.second_currency_wrapper {
+  display: flex;
+  flex-direction: row;
```
Comment (sumitd94) [inline components/exchange-rate-row/exchange-rate-row.module.css:41]: NIT: not needed, by default the direction is assumed as `row`

File: components/exchange-rate-row/exchange-rate-row.module.css [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+  box-shadow: 0px 3px 15px rgba(0, 0, 0, 0.2);
+}
+.exchange_rate_row__arrow {
+  font-size: 30px;
+  margin: auto 0;
+  color: #4a47a3;
+}
+.first_currency_wrapper {
+  display: flex;
+  flex-direction: row;
```
Comment (sumitd94) [inline components/exchange-rate-row/exchange-rate-row.module.css:24]: NIT: not needed, by default the direction is assumed as `row`

File: components/exchange-rate-row/exchange-rate-row.module.css [hunk 7]
```diff
@@ -0,0 +1,77 @@
+.exchange_rate_row {
+  font-weight: 500;
+  display: flex;
+  flex-direction: row;
```
Comment (sumitd94) [inline components/exchange-rate-row/exchange-rate-row.module.css:4]: NIT: not needed, by default the direction is assumed as `row`

File: components/custom-button/custom-button.style.js [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+  background-color: #ee1a75;
+  border: none;
+  color: white;
+  &:hover {
+    /* background-color: white;
+    border: 1px solid #4285f4;
+    color: #4285f4;
+    transform: translate(0, -10%);
+    transition: all 0.2s; */
+  }
```
Comment (sumitd94) [inline components/custom-button/custom-button.style.js:29]: We can remove this if not needed.

**Review by sumitd94 — CHANGES_REQUESTED** [review]

File: pages/currency-exchange/currency-exchange.module.css (hunk 1 above)
Comment (Akashdeep-Patra) [reply pages/currency-exchange/currency-exchange.module.css:7]: we would probably need, I am still testing the views, this is just the initial mr to show how this looks at this moment

File: pages/currency-exchange/index.js (hunk 2 above)
Comment (Akashdeep-Patra) [reply pages/currency-exchange/index.js:4]: didn't get exactly get it sumit?

File: styles/Home.module.css (hunk 3 above)
Comment (Akashdeep-Patra) [reply styles/Home.module.css:32]: yes will do that

File: components/exchange-rate-row/index.js (hunk 4 above)
Comment (Akashdeep-Patra) [reply components/exchange-rate-row/index.js:44]: Hi Sumit already have a similar solution, I posted this Mr just to get a fair view of the required changes, suppose we want to have a different style for the right currency we might need that . if you are happy with how it looks i can optimized the code for the final version

File: components/custom-button/custom-button.style.js (hunk 8 above)
Comment (Akashdeep-Patra) [reply components/custom-button/custom-button.style.js:29]: i will implement this , just thinking about the color scheme

File: pages/currency-exchange/currency-exchange.module.css (hunk 1 above)
Comment (sumitd94) [reply pages/currency-exchange/currency-exchange.module.css:7]: Sure, in that case can you please add a WIP tag to this PR?

File: pages/currency-exchange/index.js (hunk 2 above)
Comment (sumitd94) [reply pages/currency-exchange/index.js:4]: this import statement should be coming after line no 1. (good practise)

File: components/exchange-rate-row/index.js (hunk 4 above)
Comment (sumitd94) [reply components/exchange-rate-row/index.js:44]: Oh ok, got it. Looks good to me, let others put in their views as well.

**Review by sumitd94 — APPROVED** [review]
Great job, Looks good.
